pkg/yamlmeta/deep_copy.go

```diff
@@ -3,96 +3,96 @@
 
 package yamlmeta
 
-func (n *DocumentSet) DeepCopyAsNode() Node { return n.DeepCopy() }
```

Comment:
i would typically recommend isolating renaming work into separate commits from feature work. it's harder to grok if something meaningful changed.

Reply:
The additional effort to separate portions of a commit that are renames vs. semantic changes is real. It notably took us some time to do that with this change set.